GoogleForCreators/web-stories-wp
# Dashboard: Converted BlinkOn animation to new standard

## Summary

This PR converts our original BlinkOn animation into a new animation part.  I also added in storybook examples for WAAPI and AMP animations.

## Relevant Technical Choices

- I created `SimpleAnimation` as a means to share basic animation implementations across multiple animations parts while still having the ability to customize implementations per animation if needed.
- Also implemented an updated keyframes pattern so dynamic keyframes can be supported for any animation that requires it (BlinkOn being one of them).
- Renamed `animationParts` folder to `parts` and `animationOutputs` to `outputs`.

## To-do

- Convert the remaining animation parts.

## User-facing changes

- None

## Testing Instructions

- Open up Storybook and navigate to: `Dashboard -> Animations -> BlinkOn` and make sure both `Default` and `AMP Story` animations work.

---

Part of #1885 

## Screenshots

BlinkOn - WAAPI
![waapi_blinkon](https://user-images.githubusercontent.com/40646372/82507903-5c372800-9ab8-11ea-8927-3b960f5c482c.gif)

BlinkOn - AMP
![amp_blinkon](https://user-images.githubusercontent.com/40646372/82507907-5fcaaf00-9ab8-11ea-8cb4-f799d72a47f7.gif)
